packages/nimble-components/src/wafer-map/tests/utilities.ts

```diff
@@ -201,6 +215,27 @@ export function getWaferMapMockComputations(
     };
     return waferMapMock as WaferMap;
 }
+export function getWaferMapMockComputationsExperimental(
+    diesTable: Table = getWaferMapDiesTable(),
+    originLocation: WaferMapOriginLocation,
+    canvasWidth: number,
+    canvasHeight: number,
+    validity: WaferMapValidity = {
+        invalidGridDimensions: false,
+        invalidDiesTableSchema: false
+    }
+): Pick<
```

Comment:
Use the same patterns as pushed to thee other PPR to reduce `as` casts, i.e. return WaferMap instead of Pick<WaferMapp> see: https://github.com/ni/nimble/pull/1916/commits/15ad11c33712a29e11d60ba44611cac969f0d637#diff-f8244bcd1e327bb58976a036662c316f8d3bf6b6e2f3428ebc374ca882985d10R163

And discussion here:
https://github.com/ni/nimble/pull/1916#discussion_r1537864808